airflow/models.py

```diff
@@ -2609,7 +2611,10 @@ def get_extra_links(self, dttm, link_name):
             one of the options specified in `extra_links`
         :return: A URL
         """
-        pass
+
+        if link_name in type(self).extra_link_functions:
```

Comment:
Why get the `type(self)` functions and not the instance version? Using the instance version will still allow monkey patching, and I think is clearer

Reply:
fixed.